modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -1246,7 +1246,7 @@ private static void addNodeTraverseChildrenMethod(
             );
         } else if (!polymorphicFieldsByExtension.isEmpty()) {
             assert polymorphicTypeIdFieldDef != null : schemaClass.getName();
-            assert isPolymorphicId(schemaFields.get(0)) : schemaClass.getName();
+            assert schemaFields.stream().anyMatch(ConfigurationUtil::isPolymorphicId) : schemaClass.getName();
```

Comment:
Please modify the documentation for `org.apache.ignite.configuration.annotation.PolymorphicConfig`.
And also please add text to the assertion, for example: `"Missing field with @PolymorphicId in " + schemaClass.getName();`

Reply:
Done